projects/aca-content/src/lib/components/search/search-input-control/search-input-control.component.scss

```diff
@@ -6,6 +6,7 @@ $top-margin: 12px;
   font-size: 16px;
   padding-left: 15px;
   box-sizing: border-box;
+  margin-bottom: 12px !important;
```

Comment:
Is the `important` required?

Reply:
seems to be needed, as some other styles messing this button
![image](https://user-images.githubusercontent.com/503991/218819688-5fe9874d-5294-434a-a8e6-59b4c0b01928.png)